Subject: Avoid downloading extra sound files
From: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Forwarded: no
Last-Update: 2010-02-03

Asterisk configures several sound files to be installed that are not
included in the distribution tarball. Those files are downloaded by the
'install' target.

The exact files to be downloaded is configurable. Here we change the
default to avoid downloading any. We believe those should be part of
a separate source package (as they rarely change, and have their own
versioning).

--- a/sounds/sounds.xml
+++ b/sounds/sounds.xml
@@ -6,7 +6,6 @@
 		<member name="CORE-SOUNDS-EN-ALAW" displayname="English, a-Law format">
 		</member>
 		<member name="CORE-SOUNDS-EN-GSM" displayname="English, GSM format" >
-			<defaultenabled>yes</defaultenabled>
 		</member>
 		<member name="CORE-SOUNDS-EN-G729" displayname="English, G.729 format">
 		</member>
@@ -75,7 +74,6 @@
 	</category>
 	<category name="MENUSELECT_MOH" displayname="Music On Hold File Packages" positive_output="yes">
 		<member name="MOH-OPSOUND-WAV" displayname="opsound.org Music On Hold Files, WAV format" >
-			<defaultenabled>yes</defaultenabled>
 		</member>
 		<member name="MOH-OPSOUND-ULAW" displayname="opsound.org Music On Hold Files, mu-Law format" >
 		</member>
